src/metabase/search/config.clj

```diff
@@ -85,7 +85,9 @@
     :text                5
     :mine                1
     :exact               5
-    :prefix              0}
+    :prefix              0
+    ;; TODO figure out a better way to blend these scorers with wildly different scale.
+    :rrf                 5000}
```

Comment:
This weight is wacky because the rrf scores are small, so they need to be scaled so the other terms don't completely dominate.

Reply:
Of note: the large weight here means that the rrf term tends to dominate. We could lower it and get more input of the metadata scorers, but this was close to the lowest value that didn't cause a bunch of test failures due to things getting reranked, and seemed like a conservative starting point.